tests/unit/10_build-env.bats

```diff
@@ -44,6 +44,20 @@ teardown() {
   assert_success
 }
 
+@test "(build-env) buildpack if Dockerfile present" {
```

Comment:
I don't think this does what you think it does. Let me know what we're trying to do here and I'll slap something together.